opentracing-jaxrs2-itest/common/src/main/java/io/opentracing/contrib/jaxrs2/itest/common/AbstractServerTest.java

```diff
@@ -27,7 +30,7 @@ public void testServerStandardTags() throws Exception {
             .get();
         response.close();
 
-        Assert.assertEquals(1, mockTracer.finishedSpans().size());
+        Assert.assertEquals(2, mockTracer.finishedSpans().size());
```

Comment:
Could you please remove server entities from `/clientTracingChaining` and `"/hello/{id}` so the tests remain as they are?